### PR TITLE
Add strict connection cloud settings

### PR DESCRIPTION
**Pull request by bramkragten**
CleanShot 2024-04-22 at 16 57 06

For https://github.com/home-assistant/core/pull/115814

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [X] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/translations/en.json [hunk 1]
```diff
@@ -3836,6 +3836,14 @@
               "advanced_options": "Advanced options",
               "external_activation": "Allow external activation of remote control",
               "external_activation_secondary": "Allows you to turn on remote control from your Nabu Casa account page, even if you're outside your local network",
+              "strict_connection": "Restrict access to logged in users",
+              "strict_connection_secondary": "When a user is not logged in to your Home Assistant instance, they will not be able to access your instance remotely",
+              "strict_connection_mode": "Mode",
+              "strict_connection_modes": {
+                "disabled": "Disabled",
+                "static_page": "Error page",
```
Comment (edenhaus) [inline src/translations/en.json:3844]:
```suggestion
                "static_page": "Static page",
```

I would call it static page to be consistent with the non-cloud wording

Comment (matthiasdebaat) [reply src/translations/en.json:3844]: Or Unauthorized page for both? For Static page you have to known a bit about development.

Comment (edenhaus) [reply src/translations/en.json:3844]: I'm not sure if we should use `Unauthorized` as strict connection is before our authorization layer.

**Review by piitaya — DISMISSED** [review]
Code wise, it's ok for me. Let's discuss the wording with the product team.

File: src/translations/en.json (hunk 1 above)
Comment (matthiasdebaat) [reply src/translations/en.json:3844]: Couple of alternatives:
- Block page
- Guard page
- Safeguard page

Comment (frenck) [reply src/translations/en.json]: As we concluded on Discord: Guard page.

**Review by piitaya — APPROVED** [review]

**Pull request merged**